cloudify_aws/rds/tests/test_subnet_group.py

```diff
@@ -83,7 +87,7 @@ def test_create_raises_UnknownServiceError(self):
                                 resource_config=resource_config, iface=None)
 
         self.assertEqual(
-            str(error.exception),
+            text_type(error.exception),
```

Comment:
Same question. No change is necessary, just curious.

Reply:
Same as above